backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Director.kt

```diff
@@ -0,0 +1,7 @@
+package com.pk.zbtz.zbtzbackend.domain
+
+data class Director(
+    val id: Long,
+    val firstName: String,
+    val lastName: String
```

Comment:
I would add image here (we should be able to show his face somehow). Birthday would also be nice to have

Reply:
Added url with photo